ports/stm32/rfcore.c

```diff
@@ -556,11 +555,15 @@ void rfcore_ble_init(void) {
 
     // Clear any outstanding messages from ipcc_init
     tl_check_msg(&ipcc_mem_sys_queue, IPCC_CH_SYS, NULL);
-    tl_check_msg_ble(&ipcc_mem_ble_evt_queue, NULL);
+    tl_check_msg(&ipcc_mem_ble_evt_queue, IPCC_CH_BLE, NULL);
```

Comment:
is this line even needed anymore, now that events are handled immediately in the RX IRQ?